src/amuse/rfi/channel.py

```diff
@@ -806,6 +808,8 @@ def split_message(self, call_id, function_id, call_count, dtype_to_arguments, en
         self._communicated_splitted_message = True
         self._merged_results_splitted_message = dtype_to_result
 
+    def makedirs(self,directory):
+        os.makedirs(directory)
```

Comment:
not sure why - could we not just call os.makedirs directly as before?

Reply:
its in the base class to be overridden, ie the default implementation (for local codes) of course needs to call makedirs, but remotely you do something else (distributed or sockets channel)